ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -283,14 +283,21 @@ Item {
         id: fetchMoreMessagesButtonComponent
         Item {
             id: wrapper
+            property int gapNowAndOldest: nextMessageIndex > -1 ? (Date.now() / 1000 - nextMsgTimestamp / 1000) : 0
+            property int gapNowAndJoined: Date.now() / 1000 - +chatsModel.channelView.activeChannel.joined / 1000
+
+            visible: {
+                return gapNowAndOldest < Constants.maxNbDaysToFetch * Constants.fetchRangeLast24Hours
+                        && gapNowAndJoined > Constants.maxNbDaysToFetch * Constants.fetchRangeLast24Hours
```

Comment:
Could maybe pull this out in to a variable?

Reply:
Fixed